### ISSUE TITLE
`toLowerCase` undefined when use command `bridgeall <PID>`

### ISSUE THREAD

**Issue by limeless** [matrix-discord/mx-puppet-discord#182]
log shows 
```
2021-11-06T08:33:23.831550921Z app[web.1]: Nov-6 08:33:23.831 [MatrixEventHandler] error: Error handling appservice room.event TypeError: Cannot read property 'toLowerCase' of undefined
2021-11-06T08:33:23.831596450Z app[web.1]:     at Commands.<anonymous> (/opt/mx-puppet-discord/build/Commands.js:223:54)
2021-11-06T08:33:23.831603130Z app[web.1]:     at Generator.next (<anonymous>)
2021-11-06T08:33:23.831607740Z app[web.1]:     at /opt/mx-puppet-discord/build/Commands.js:20:71
2021-11-06T08:33:23.831612020Z app[web.1]:     at new Promise (<anonymous>)
2021-11-06T08:33:23.831627880Z app[web.1]:     at __awaiter (/opt/mx-puppet-discord/build/Commands.js:16:12)
2021-11-06T08:33:23.831632480Z app[web.1]:     at Commands.commandBridgeAll (/opt/mx-puppet-discord/build/Commands.js:217:16)
2021-11-06T08:33:23.831636820Z app[web.1]:     at BotProvisioner.<anonymous> (/opt/mx-puppet-discord/node_modules/mx-puppet-bridge/lib/src/botprovisioner.js:180:59)
2021-11-06T08:33:23.831641400Z app[web.1]:     at Generator.next (<anonymous>)
2021-11-06T08:33:23.831645679Z app[web.1]:     at fulfilled (/opt/mx-puppet-discord/node_modules/mx-puppet-bridge/lib/src/botprovisioner.js:17:58)
2021-11-06T08:33:23.831650129Z app[web.1]:     at runMicrotasks (<anonymous>)
```

Using docker image from docker hub with `link user <usertoken>` 
deploy via `$ dokku git:from-image sorunome/mx-puppet-discord:latest`

maybe it's a issue with `mx-puppet-bridge` ?

**Comment by limeless** [comment]
My bad after `help bridgeall` I've figured it is `bridgeall <PID> <1/0>`

but error message is bit confusing. because it's trying to convert `null` to `toLowerCase()` which it should be handled.

Thanks.

**Comment by limeless** [comment]
fix #183
